README.rst

```diff
@@ -38,7 +38,7 @@ for instance nanoseconds it will round it to microseconds.
 Documentation
 -------------
 
-Currently there are four parsing methods available.
+Currently there are five parsing methods available.
```

Comment:
Better yet, skip the number for when it changes again next time! We can also skip "currently", as we're not documenting how things were or will be:

```suggestion
The following parsing methods are available.
```

ref: https://twitter.com/mikepope/status/1341786385812844549

Otherwise looks good!